integrations/klaviyo/lib/index.js

```diff
@@ -132,7 +132,8 @@ Klaviyo.prototype.orderCompleted = function(track) {
     $value: track.revenue(),
     Categories: products.categories,
     ItemNames: products.names,
-    Items: products.items
+    Items: products.items,
+    total: track.total()
```

Comment:
I just read through the rest of the code. It seems like to send `total` work, all we have to do is to remove 'total' from whitelist array below. (Line number 146-147)